modules/qna/src/views/full/utils/qnaList.utils.ts

```diff
@@ -47,7 +47,7 @@ export const itemHasError = (qnaItem: QnaItem, currentLang: string): string[] =>
   if (!hasPopulatedLang(data.questions)) {
     errors.push(lang.tr('module.qna.form.missingQuestion'))
   }
-  if (!hasPopulatedLang(data.answers) && !data.redirectFlow && !data.redirectNode) {
+  if (!hasPopulatedLang(data.answers) && !Object.values(data.contentAnswers).reduce((acc, arr) => [...acc, ...arr], []).length && !data.redirectFlow && !data.redirectNode) {
```

Comment:
You should check if data.contentAnswers is undefined also before reading its values.

Also, since the list is getting kind of long, you could extract the logic otuside the condition, ex `hasContentAnswer`